test-requirements.txt

```diff
@@ -12,4 +12,4 @@ pytest-xdist
 pexpect
 git+https://github.com/ansible-network/pytest-ansible-network-integration.git
```

Comment:
Unrelated, but can you please fix the pytest-ansible requirements in this collection as well?